tests/auto/rt_auto_20210223020202.log

```diff
@@ -0,0 +1,81 @@
+INFO:MAIN:Starting Script
```

Comment:
Please delete these rt_auto_YYYYMMDD*.log files. They should not be in the repository. These are just verbose stdout/stderr from a script.

Reply:
Being that everything is automated away from the watchful eyes of code managers, I thought it helpful to have a verbose log, that includes stdout items as well as specific items for where in the script it is in case of failure. I think its important to have something. I will find another way to get that information without committing it in a PR repo.

I think the API allows me to post a comment, but that comment will be large. I can try going that route if its sufficient enough?

Reply:
There are two rt_auto files. They have the same root name but one (tests/auto/rt_auto_20210223031812.log) is 1.8MB and contains totally different information it seems.